### DISCUSSION TITLE
Explore Notification Center integration like Replit

### DISCUSSION THREAD

**Discussion by jonbell-lot23** [RecordReplay/devtools#2323] [Ideas]
I just used Replit and they asked for permission to connect to macOS's Notification Center.

Worth considering alongside our email integration.

**Comment by jasonLaster** [comment]
Lets add this as a discussion, since it's something that will probably bake for a little while as we figure it out with Knock or Courier

**Comment by jonbell-lot23** [comment]
Bingo, was just coming here to say the same thing.

**Comment by jonbell-lot23** [comment]
March 15 2022 update:

As we see usage increase, ideas like this are definitely coming more into focus. The more people we have counting on us, and the more replays they're tracking, the more features like this will help.
